src/stampit.js

```diff
@@ -207,10 +203,9 @@ stampit = function stampit(options) {
      * @param {[Function]|...Function} factories Stampit factories.
      * @return {Function} A new stampit factory composed from arguments.
      */
-    compose: function (factories) {
-      var args = isArray(factories) ? factories : slice(arguments);
-      args = [factory].concat(args);
-      return compose(args);
+    compose(...factories) {
+      let args = [factory].concat(factories);
```

Comment:
Sorry, I read your question wrong. Yes, I am making this change to all relevant locations.